salt/modules/mac_ports.py

```diff
@@ -323,7 +323,13 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
 
     formulas_array = []
     for pname, pparams in six.iteritems(pkg_params):
-        formulas_array.append(pname + (pparams or ''))
+        formulas_array.append(pname)
+
+        if 'version' in pparams:
```

Comment:
you should do a `if pparams and 'version' in pparams:`

because it is possible that pparams is a None value.